src/components/map/map-component.tsx

```diff
@@ -331,13 +331,17 @@ export class MapComponent extends BaseComponent<IProps, IState>{
     const { samplesCollectionsStore } = this.stores;
     const { volcanoLat, volcanoLng } = this.stores.simulation;
     const riskItems: React.ReactElement[] = [];
+    const tabIndex = this.stores.uiStore.currentHistogramTab;
+    const histogramCharts = this.stores.chartsStore.charts.filter(chart => chart.type === "histogram");
+    const currentChart = histogramCharts && histogramCharts[tabIndex];
+    const chartName = currentChart && currentChart.title ? currentChart.title : undefined;
```

Comment:
```suggestion
    const chartName = currentChart?.title || undefined;
```

Reply:
I believe we need to move to typescript 3.7 to support optional chaining (GeoCode is on TS 3.6).  Which is a good reminder that we should look at updating GeoCode dependencies (not sure when that was done last).